### ISSUE TITLE
Ability - Deja Vu

### ISSUE THREAD

**Issue by PressPlaySweden** [steven4547466/AdvancedSubclassing#36]
**Is your feature request related to a problem? Please describe.**
A new ability, player would type a command and set a point, then type another command to teleport back to that point.
This would need a cooldown for the point placing and also one for the teleporting.

**Describe the solution you'd like**
I would like the player to type a command on their current position, this would place a save point. Then the player is able to type another command to teleport back to that save point. The player can place a new save point after a cooldown, if their already is a save point, it would get replaced with the new one. 

**Describe alternatives you've considered**
N/A

**Additional context**
In SCP Containment breach, the main character you play as is a SCP that can see many futures, so every time you die, you go back to the start. I would like to create him as a class. And the first ability needed is the go back in time ability.

**Comment by steven4547466** [comment]
Do you want stuff like health armor and inventory at the time to be saved?

**Comment by PressPlaySweden** [comment]
Yes, if that is possible. Make it so his past inventory is saved. So when he teleports back, he recives what he had when he was at that point.
